.github/workflows/run-ut-and-coverage.yml

```
@@ -12,6 +12,11 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha || github.event.merge_group.head_sha || github.sha }}
           fetch-depth: 0
+          # recent actions/checkout v4 releases refuse fork-PR checkouts under
+          # pull_request_target by default. Running the PR's test suite in
+          # this context is this workflow's long-standing design; opt in
+          # explicitly to restore it.
+          allow-unsafe-pr-checkout: true
```

actionlint (1.7.12)

[error] input "allow-unsafe-pr-checkout" is not defined in action "actions/checkout@v4". available inputs are "clean", "fetch-depth", "fetch-tags", "filter", "github-server-url", "lfs", "path", "persist-credentials", "ref", "repository", "set-safe-directory", "show-progress", "sparse-checkout", "sparse-checkout-cone-mode", "ssh-key", "ssh-known-hosts", "ssh-strict", "ssh-user", "submodules", "token"

(action)